README.md

```diff
@@ -48,8 +48,10 @@ The driver can be selected via the `--vm-driver=xxx` flag of `minishift start`.
 [docker machine drivers](./docs/docker-machine-drivers.md) for more details on supported drivers
 and how to install them.
 
-**Note**: For most hyervisor VT-x/AMD-v virtualization must be enabled in BIOS. For Hyper-V, however,
+**Note**:
+- For most hyervisor VT-x/AMD-v virtualization must be enabled in BIOS. For Hyper-V, however,
 it needs to be disabled.
+- We recommend to use `Virtualbox 5.1.12` for Windows to avoid [known issue](./docs/troubleshooting.md/#troubleshooting.md#error-getting-state-for-host-machine-does-not-exist)
```

Comment:
We might be confusing this version issue with https://github.com/minishift/minishift/issues/236.

Even, @agajdosi reported same here https://github.com/minishift/minishift/issues/159#issuecomment-269957348.

For me I couldn't reproduce issue even with `5.0.30` if running from `%USERPROFILE%` containiing drive.

This forcing me to think whether we require this note or not. 

However, I would suggest to have proper testing around `%USERPROFILE%` and check.